app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -606,6 +608,10 @@ $after-button-padding-left: govuk-spacing(4);
 // Styles for search toggle button.
 .gem-c-layout-super-navigation-header__search-toggle-button {
   background: none;
+
+  &--blue-background {
```

Comment:
Just a small suggestion - I don't think we have any style guidelines on this, but these types of selectors can be a bit tricky to search for. We could change it to `gem-c-layout-super-navigation-header__search-toggle-button--blue-background` though I realise that might be a bit verbose so no worries if not.

Reply:
Thanks @jon-kirwan - good suggestion, will do this and then merge.